### PR TITLE
issue #70: updated the readme with the current env var requirements

### LINKED ISSUE #70: Update documentation with new env var requirements
The documentation is outdated. There are new environment variables requirements which have not been documented.

### PR DESCRIPTION
**Pull request by k-allagbe**
#70
issue-70-update-env-vars-in-readme

I updated the env var requirements in the readme. Now, only some of the env variables are required to be set. Make sure to set them properly before deploying.

Closes #70

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,158 +57,167 @@ To run the Quart application correctly, it requires some environment variables t
...
+#### MEMBRANE_SESSION_COOKIE_SECURE
 
 - **Description:** Indicates if the session cookie should be secure.
 - **Example:** `MEMBRANE_SESSION_COOKIE_SECURE=true`
 
-### MEMBRANE_SESSION_TYPE
+#### MEMBRANE_SESSION_TYPE
 
 - **Description:** Specifies the storage for session data. Options: 'filesystem', 'redis', 'memcached', etc.
 - **Example:** `MEMBRANE_SESSION_TYPE=null`
```
Comment (rngadam) [inline README.md:128]: Can you go back through the environment variable and add a 

**Reference: https://flask-session.readthedocs.io/en/latest/config.html

with a link to the subsystem doc that uses this value? It occurs to me the description is not sufficient to decide the value to use and I was unaware that null was a valid value for example.

**Review by rngadam — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (k-allagbe) [reply README.md:128]: Done

Comment (rngadam) [reply README.md:128]: Repeat adding reference for all variables that feed dependency libraries.

**Review by rngadam — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (k-allagbe) [reply README.md:135]: Done

**Review by rngadam — APPROVED** [review]

**Pull request merged**